recipes/fractal-server/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "fractal-server" %}
+{% set version = "1.3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fractal_server-{{ version }}.tar.gz
+  sha256: 826c3fdc075073d3838c4a94d28d0f2680594171fe85d533a78905a1f280e1c0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - fractalctl = fractal_server.__main__:run
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0
+    - python-dotenv >=0.20.0,<0.21.0
+    - fastapi >=0.95.0,<0.96.0
+    - sqlmodel >=0.0.8,<0.0.9
+    - aiosqlite >=0.17.0,<0.18.0
+    - fastapi-users >=10.1.0,<11.0.0
+    - alembic >=1.9.1,<2.0.0
+    - uvicorn >=0.20.0,<0.21.0
+    - sqlalchemy >=1.4.0,<2.0.0
+    - sqlalchemy-utils >=0.38.3,<0.39.0
+    - greenlet *
+    - pydantic >=1.10.8,<2
+  run_constrained:
+    - clusterfutures >=0.5.0,<0.6.0
+    - cloudpickle >=2.2.1<2.3.0
+    - asyncpg >=0.27.0,<0.28.0
+    - psycopg2 >=2.9.5,<3.0.0
+    - gunicorn >=20.1.0,<21.0.0
+
+test:
+  imports:
+    - fractal_server
+  commands:
```

Comment:
Could you test the cli under the test/commands section? Just `- fractalctl --help` or similar.

Reply:
Of course, thanks! I fixed it with https://github.com/conda-forge/staged-recipes/pull/24119/commits/3e443d725ced7f3d422fc6018ce1508be5af4153.